ginga/util/io_fits.py

```diff
@@ -357,7 +367,9 @@ def close(self):
         self.hdu_db = {}
         self.extver_db = {}
         self.info = None
+        fits_f = self.fits_f
```

Comment:
How is this different from below? Just curious.
```python
self.fits_f.close()
self.fits_f = None
```

Reply:
If you try to `close` and it raises an exception, the object will not be dereferenced and garbage collected. I want to make sure that it is dereferenced no matter what.